### PR TITLE
Remove implicit parameters from Sublist.Propositional.Properties

### PR DESCRIPTION
**Pull request by MatthewDaggitt**
All the knock on effects seem to be internal to the module only. Cross fingers will pass CI.

### CODE REVIEW

**Comment by jamesmckinna** [comment]
Thanks @MatthewDaggitt for this.
Given everything you've written so far about parametrisation of `Properties` modules, this seems to be the 'right' path to take regarding the overall `module` structure.

File: src/Data/List/Relation/Binary/Sublist/Propositional/Properties.agda [hunk 1]
```diff
@@ -31,28 +30,29 @@ open import Relation.Unary using (Pred)
...
 
-open SetoidProperties (setoid A) public
-  hiding (map⁺; ⊆-trans-idˡ; ⊆-trans-idʳ; ⊆-trans-assoc)
+module _ {A : Set a} where
+  open SetoidProperties  (setoid A) public
+    hiding (map⁺; ⊆-trans-idˡ; ⊆-trans-idʳ; ⊆-trans-assoc)
 
 map⁺ : ∀ {as bs} (f : A → B) → as ⊆ bs → map f as ⊆ map f bs
-map⁺ {B = B} f = SetoidProperties.map⁺ (setoid A) (setoid B) (cong f)
+map⁺ {B = B} f = SetoidProperties.map⁺ (setoid _) (setoid _) (cong f)
```
Comment (jamesmckinna) [inline src/Data/List/Relation/Binary/Sublist/Propositional/Properties.agda:44]: Is the `{B = B}` binding redundant now?

File: src/Data/List/Relation/Binary/Sublist/Propositional/Properties.agda [hunk 2]
```diff
@@ -31,28 +30,29 @@ open import Relation.Unary using (Pred)
 
 private
   variable
-    b ℓ : Level
-    B : Set b
+    a ℓ : Level
+    A B : Set a
```
Comment (jamesmckinna) [inline src/Data/List/Relation/Binary/Sublist/Propositional/Properties.agda:34]: Elsewhere, eg. `Data.List.Relation.Binary.Sublist.Setoid.Properties`, the `variable` block is enriched with bindings for 'typical' `List A` arguments. Can the same be done here, to simplify the binding prefixes in eg. L43 below, and elsewhere?

**Review by jamesmckinna — APPROVED** [review]
Looks good to me, modulo passing the tests. 
Some suggested tweaks, but they are cosmetic, and could be fixed up once we revisit this issue (`module` paramterisation vs. `variable` declarations) more systematically throughout the library.

File: src/Data/List/Relation/Binary/Sublist/Propositional/Properties.agda (hunk 1 above)
Comment (MatthewDaggitt) [reply src/Data/List/Relation/Binary/Sublist/Propositional/Properties.agda]: Yup!

File: src/Data/List/Relation/Binary/Sublist/Propositional/Properties.agda (hunk 2 above)
Comment (MatthewDaggitt) [reply src/Data/List/Relation/Binary/Sublist/Propositional/Properties.agda:34]: Yup, nice idea!

**Comment by jamesmckinna** [comment]
Looking good, indeed better!
Happy to merge once tests pass.

**Pull request merged**